mods/fix-qt-scrollbar-grippers.wh.cpp

```diff
@@ -0,0 +1,87 @@
+// ==WindhawkMod==
+// @id              fix-qt-scrollbar-grippers
+// @name            Fix Qt scrollbar grippers
+// @description     Fixes scrollbar grippers in Qt 6.1+
+// @version         1.0
+// @author          kawapure
+// @github          https://github.com/kawapure
+// @twitter         https://twitter.com/kawaipure
+// @homepage        https://kawapure.github.io
+// @include         ida64.exe
```

Comment:
Was it meant to only target IDA?

Reply:
IDA was the application that I tested against, but it should work for any Qt 6.1+ application using native theme scrollbar controls. I thought that it would be obvious to any end user to specify custom targets. Should this be clarified in the README?

Reply:
Yes, better clarify it, something like:
https://github.com/ramensoftware/windhawk-mods/blob/a480102c52e3cd16e1cea214913e1395155c82c0/mods/ignore-focus-loss.wh.cpp#L20-L23

Reply:
I copied that clarification.